index.js

```diff
@@ -15,6 +15,38 @@ const sortObject = onObject(sortObjectKeys())
 const sortURLObject = sortObjectBy(['type', 'url'])
 const sortAuthorObject = sortObjectBy(['name', 'email', 'url'])
 const sortDirectories = sortObjectBy(['lib', 'bin', 'man', 'doc', 'example'])
+const sortProperty = (property, over) => object =>
+  Object.assign(object, { [property]: over(object[property]) })
+
+const sortGitHooks = sortObjectBy(
+  // https://git-scm.com/docs/githooks
+  [
+    'applypatch-msg',
+    'pre-applypatch',
+    'post-applypatch',
+    'pre-commit',
+    'pre-merge-commit',
+    'prepare-commit-msg',
+    'commit-msg',
+    'post-commit',
+    'pre-rebase',
+    'post-checkout',
+    'post-merge',
+    'pre-push',
+    'pre-receive',
+    'update',
+    'post-receive',
+    'post-update',
+    'push-to-checkout',
+    'pre-auto-gc',
+    'post-rewrite',
+    'rebase',
+    'sendemail-validate',
+    'fsmonitor-watchman',
+    'p4-pre-submit',
+    'post-index-change',
+  ],
```

Comment:
You prefer keep this list here, or a separate package I build https://github.com/fisker/git-hooks-list ?

Reply:
Yes I was thinking we could start moving a lot of this stuff out to dependencies. I'm happy to add this dependency 👍

Reply:
done.